LINK/etc/default/evm

```diff
@@ -25,6 +25,9 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 # Location of certificates and provider keys
 export KEY_ROOT=/var/www/miq/vmdb/certs
 
+export APPLIANCE_REPO_DIRECTORY=/opt/manageiq/manageiq-appliance
```

Comment:
The problem is the appliance word is overloaded here.  I thought APPLIANCE_REPO_DIRECTORY was more clear it was the manageiq-appliance github repo's files.